src/main/java/seedu/recipe/logic/commands/ImportCommand.java

```diff
@@ -0,0 +1,63 @@
+package seedu.recipe.logic.commands;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.stage.Stage;
+import seedu.recipe.commons.core.LogsCenter;
+import seedu.recipe.commons.exceptions.IllegalValueException;
+import seedu.recipe.logic.commands.exceptions.CommandException;
+import seedu.recipe.model.Model;
+import seedu.recipe.model.recipe.Recipe;
+import seedu.recipe.storage.ImportManager;
+
+/**
+ * Represents the command used to import RecipeBook JSON file from another file location into this RecipeBook app.
+ * This is deliberately triggered and created by a UI interaction using a FileWindow, as the FileWindow guarantees
+ * the filepath passed to the app by the user is not malformed.
+ */
+public class ImportCommand extends Command {
+    public static final String EMPTY_COMMAND = "No file was selected.";
```

Comment:
Eventually maybe can shift all the messages into the `commons/core/Messages.java file` since they are user visible messages!

Reply:
V1.4 CQ note